scripts/cleanup_submissions/README.md

```diff
@@ -0,0 +1,9 @@
+## Delete unused workflows for submissions in a workspace
+This workflow can be run from a FireCloud workspace, or locally, to delete workflows that aren't used by the data model (entity attributes) or workspace attributes.
+
+### Inputs:
+  `File svcActKeyJson` - Service account credentials JSON, needs write access to the workspace bucket containing submissions.
```

Comment:
I wonder if we should point people to the register script and say specifically that the service account should be added to the workspace as a Writer or Owner

Reply:
Absolutely--that is the true requirement.